app/models/beer.rb

```diff
@@ -32,38 +32,92 @@ def self.highest_rated
     end
   end
 
-  #create methods to be able to sort by category(name, abv, rating)
-  def self.sort_beers_by_name_asc
-    #list all beers by category, give the user the option for ASC or DESC
-    self.order(:name)
-  end
-
-
+  
+  
   def self.sort_beers_option
     choices = {'by name' => 1, 'by abv' => 2}
     selection = PROMPT.select("How would you like to sort?", choices)
-
+    
     case selection
+    when 1
+      options = {'A - Z' => 1, 'Z - A' => 2}
+      selection1 = PROMPT.select("How would you like to sort?", options)
+      case selection1
       when 1
+        self.sort_beers_by_name_asc
+      when 2
         self.sort_beers_by_name_desc
+      end
+    when 2
+      options = {'lowest abv' => 1, 'highest abv' => 2}
+      selection2 = PROMPT.select("How would you like to sort?", options)
+      case selection2
+      when 1
+        self.sort_beers_by_abv_asc
       when 2
-        self.order(abv: :desc)
+        self.sort_beers_by_abv_desc
+      end
     end
   end
 
+  def self.sort_beers_by_name_asc
+    ordered_beers = self.order(:name)
+    ordered_beers.each_with_index do |beer,index|
+      i_plus_one = index +1
+      beer_name = "Beer: #{beer.name}"
+      abv = "Abv: #{beer.abv}"
+      description = "Description: #{beer.description}"
+      puts "#{i_plus_one}."
+      puts ColorizedString[beer_name].colorize(:light_blue) 
+      puts ColorizedString[abv].colorize(:light_yellow)
+      puts ColorizedString[description].colorize(:light_white) 
+      puts " ------ "
+    end
+  end
+  
   def self.sort_beers_by_name_desc
-    binding.pry
-    self.order(name: :desc)
+    ordered_beers = self.order(name: :desc)
+    ordered_beers.each_with_index do |beer,index|
+      i_plus_one = index +1
+      beer_name = "Beer: #{beer.name}"
+      abv = "Abv: #{beer.abv}"
+      description = "Description: #{beer.description}"
+      puts "#{i_plus_one}."
+      puts ColorizedString[beer_name].colorize(:light_blue) 
+      puts ColorizedString[abv].colorize(:light_yellow)
+      puts ColorizedString[description].colorize(:light_white) 
+      puts " ------ "
+    end
   end
 
   def self.sort_beers_by_abv_asc
-    #list all beers by category, give the user the option for ASC or DESC
-    self.order(:abv)
+    ordered_beers = self.order(:abv)
+    ordered_beers.each_with_index do |beer,index|
+      i_plus_one = index +1
+      beer_name = "Beer: #{beer.name}"
+      abv = "Abv: #{beer.abv}"
+      description = "Description: #{beer.description}"
+      puts "#{i_plus_one}."
+      puts ColorizedString[beer_name].colorize(:light_blue) 
+      puts ColorizedString[abv].colorize(:light_yellow)
+      puts ColorizedString[description].colorize(:light_white) 
+      puts " ------ "
+    end
   end
 
   def self.sort_beers_by_abv_desc
-    #list all beers by category, give the user the option for ASC or DESC
-    self.order(abv: :desc)
+    ordered_beers = self.order(abv: :desc)
```

Comment:
like here, in the Beer class you should just return the ordered beer objects by abv. then in your helper module clal that beer method and print it out